android-timetracker/src/com/markuspage/android/atimetracker/Tasks.java

```diff
@@ -155,15 +176,15 @@
     private Vibrator vibrateAgent;
     private ProgressDialog progressDialog = null;
     private boolean decimalFormat = false;
+
     /**
      * A list of menu options, including both context and options menu items
      */
-    protected static final int ADD_TASK = 0,
-            EDIT_TASK = 1, DELETE_TASK = 2, REPORT = 3, SHOW_TIMES = 4,
-            CHANGE_VIEW = 5, SELECT_START_DATE = 6, SELECT_END_DATE = 7,
-            HELP = 8, EXPORT_VIEW = 9, SUCCESS_DIALOG = 10, ERROR_DIALOG = 11,
-            SET_WEEK_START_DAY = 12, MORE = 13, BACKUP = 14, PREFERENCES = 15,
-            PROGRESS_DIALOG = 16;
+    protected static final int ADD_TASK = 0, EDIT_TASK = 1, DELETE_TASK = 2,
+            REPORT = 3, SHOW_TIMES = 4, CHANGE_VIEW = 5, SELECT_START_DATE = 6,
+            SELECT_END_DATE = 7, HELP = 8, EXPORT_VIEW = 9,
+            SUCCESS_DIALOG = 10, ERROR_DIALOG = 11, SET_WEEK_START_DAY = 12,
+            MORE = 13, BACKUP = 14, PREFERENCES = 15, PROGRESS_DIALOG = 16;
```

Comment:
Ah, yeah, sorry. I don't really pay attention to the formatting of the code. I usually just ignore the format of the code, but I'll try to remain respectful if I do future development.